backend/src/main/resources/db/migration/V20__Switch_tournaments_address_latlng.sql

```diff
@@ -0,0 +1,3 @@
+ALTER TABLE tournaments DROP COLUMN address;
+ALTER TABLE tournaments ADD COLUMN lat double precision;
+ALTER TABLE tournaments ADD COLUMN lng double precision;
```

Comment:
Don't you want to make the columns NOTNULL?

Reply:
tru